DependencyInjection/Configuration.php

```diff
@@ -109,6 +114,11 @@ private function addProfileSection(ArrayNodeDefinition $node)
 
     private function addRegistrationSection(ArrayNodeDefinition $node)
     {
+        if (method_exists('Symfony\Component\Form\FormTypeInterface', 'setDefaultOptions')) {
```

Comment:
This is broken, because it will use the deprecated API on Symfony 2.8, where both system exist...